shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -522,6 +527,12 @@ std::chrono::nanoseconds FlutterWindowsEngine::FrameInterval() {
   return std::chrono::nanoseconds(interval);
 }
 
+FlutterWindowsView* FlutterWindowsEngine::view(FlutterViewId view_id) const {
```

Comment:
Should we add a unit test for this now to test the implicit view ID and make sure we update everything in sync when removing that assumption?